markdown/h2m/handlers/tables.ts

```diff
@@ -17,7 +17,7 @@ export const tables: QueryAndTransform[] = [
               ? node.children
               : node
           )
-          .filter((node) => node.tagName == "tr")
+          .filter((node) => "tagName" in node && node.tagName == "tr")
```

Comment:
Is typescript clever enough to figure out doing `node?.tagName == "tr"`?

Reply:
I think that would only work if `node` is null, but here the check allows TS to infer that `node` is of type `Element`.